### ISSUE TITLE
Facebook message_reads message_echoes events not supported by Botkit

### ISSUE THREAD

**Issue by malhotraa** [howdyai/botkit#396]
The message_reads and message_echoes events aren't implemented by Botkit. I verified that they are being received correctly by the /facebook/received endpoint but no corresponding bot event is triggered.

```javascript
{ sender: { id: '1428481693844762' },
  recipient: { id: '227330390965687' },
  timestamp: 1472535337275,
  read: { watermark: 1472535290271, seq: 17971 } }

info: Got an unexpected message from Facebook:  { sender: { id: '1428481693844762' },
  recipient: { id: '227330390965687' },
  timestamp: 1472535337275,
  read: { watermark: 1472535290271, seq: 17971 } }

{ sender: { id: '227330390965687' },
  recipient: { id: '1428481693844762' },
  timestamp: 1472535356430,
  message: 
   { is_echo: true,
     mid: 'mid.1472535356404:261dbbb07e0c648089',
     seq: 17972,
     text: 'sending message' } }
```

**Comment by arunthampi** [comment]
i'll be happy to fix this as we need this asap -- but realized there are no tests for Facebook. Shall I go ahead and submit a pull request? @peterswimm @malhotraa?

**Comment by peterswimm** [comment]
@arunthampi If you submit a PR I'll definitely get our team to review it!

**Comment by arunthampi** [comment]
Done! done! https://github.com/howdyai/botkit/pull/403

**Comment by benbrown** [comment]
This is merged and will be published in 0.4!
